content/en/account_management/billing/serverless.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Serverless billing
+kind: faq
+---
+
+## Overview
+
+Purchase serverless functions on [Datadog Pro and Enterprise plans][1]. Datadog bills based on the average of your accounts hourly functions across the month.  Pro and Enterprise plans include 40 custom metrics per billed function. Contact [Sales][2] or your [Customer Success Manager][3] for more information about adding Serverless to your account.
+
+## Serverless Functions
+
+Every hour, Datadog records the number of functions that execute one or more times on your Datadog account. At the end of the month, Datadog charges by calculating the average of of the hourly number of functions recorded.
```

Comment:
Let's add a sentence like: "The billed number of functions is almost always significantly lower than the number you'll see on the Serverless page or in your AWS Console because of the average hourly metering. Functions that run infrequently are less likely to affect your bill, and sudden spikes in traffic won't cost you extra."